certbot/client.py

```diff
@@ -163,7 +163,7 @@ def perform_registration(acme, config):
 
 
 class Client(object):
-    """ACME protocol client.
+    """Certbot's client.
```

Comment:
I prefer "Certbot's client" just to disambiguate it from the acme package's client.